WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    [TestClass]
+    public class UserServiceTests : BaseServiceUnitTest
+    {
+        public static User User { get; set; }
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            Context.Add(User = new User { Id = new Guid("d58893fd-3836-4308-b840-85f4fe548264"), FirstName = "John", LastName = "Doe", Email = "JohnDoe@makinsense.com", UserName = "JohnDoe" });

```

Comment:
Mmm particularly on this line I would prefer to split in two: one for user instantiation, and the other to call the add(user).